vendor/sigs.k8s.io/controller-reconciler/pkg/reconciler/manager/gcp/redis/manager.go

```diff
@@ -256,3 +261,56 @@ func NewService(ctx context.Context) (*redis.Service, error) {
 	client.UserAgent = UserAgent
 	return client, nil
 }
+
+// --------------------- Observables -------------------------------
+
+// Observables - i
+type Observables struct {
```

Comment:
We dont want a builder pattern for observables for now.
Stick to a simple NewObservable() if possiuble